# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: docshell/base/nsDocShell.h
# Commit: ba987e21b532
# Full Hash: ba987e21b5329d07c3606a65ee0cb76e7bc99a74
# Author: Kris Maglione <maglione.k@gmail.com>
# Date: 2019-08-15 09:46:49
# Regressor Bug: 1562292
# File Overlap Count: 1
# Description:
#   Bug 1562292: Part 1d - Move OnePermittedSandboxedNavigator to BrowsingContext. r=nika
#   
#   We need to be able to check the one-permitted-sandboxed-navigator from
#   potentially-cross-process access checks in DocShell, which means it needs to
#   live on BrowsingContext rather than DocShell.
# ==============================================================================

diff -r a50f085eb3e6 -r ba987e21b532 docshell/base/nsDocShell.h
--- a/docshell/base/nsDocShell.h	Thu Aug 01 19:46:32 2019 -0700
+++ b/docshell/base/nsDocShell.h	Thu Aug 01 16:22:52 2019 -0700
@@ -1066,7 +1066,6 @@
   nsString mTitle;
   nsString mCustomUserAgent;
   nsCString mOriginalUriString;
-  nsWeakPtr mOnePermittedSandboxedNavigator;
   nsWeakPtr mOpener;
   nsTObserverArray<nsWeakPtr> mPrivacyObservers;
   nsTObserverArray<nsWeakPtr> mReflowObservers;